lib/pageflow/configuration.rb

```diff
@@ -228,6 +228,8 @@ class Configuration
     attr_accessor :default_author_meta_tag
     attr_accessor :default_publisher_meta_tag
 
+    attr_accessor :authorize_user_deletion
+
```

Comment:
Please add inline help describing the option and the lambda signature. Also add a `@since edge` doc tag. That will automatically be rewritten to the correct version when the gem is released.